templates/package/README.md

```diff
@@ -0,0 +1,21 @@
+# Zupass Package Template
+
+This directory contains template files for a new package, and a `plopfile` which can be used to generate a package.
```

Comment:
instructions on how to use plop might be nice here.

Reply:
Originally this did describe how to use it directly, but instead it tells you to use `yarn generate-package`, which wraps up a call to `yarn plop --plopfile=...`. I can see how it might be better to give a bit more background here though.